modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js

```diff
@@ -144,15 +144,38 @@ export default function DatePicker({
 
 	const [expanded, setExpanded] = useState(false);
 
+	const handleBlur = () => {
+		onBlur?.();
+
+		if (!otherProps.required) {
+			const isFill = /\d/.test(formattedDate);
```

Comment:
Hey @vngoliveira can you extract this regex `/\d/` to a const outside the DatePicker component and name it in the format `NAME_REGEX`?